package.json

```diff
@@ -49,6 +49,7 @@
     "test:visual:baseline": "./script/run-nightwatch.sh --visual --command=baseline",
     "test:visual:mobile": "./script/run-nightwatch.sh --visual --mobile",
     "test:visual:mobile:baseline": "./script/run-nightwatch.sh --visual --command=baseline --mobile",
+    "test:integration": "jest",
```

Comment:
Just thinking ahead here, but is there a way to configure the `jest` command to only run the integration test? I realize that that's _all_ there are right now, but in the future we might have Jest tests for unit tests as well. You're more than welcome to veto this suggestion as it could be considered premature optimization!

Reply:
Good point, we can specify and rename the config.